### PR TITLE
Telekinetic Slime Fix & More Cleaning

### PR DESCRIPTION
**Pull request by MrLostman**
## About The Pull Request
Fixes a bug where slimes could attack through windoors and thin windows.

## Changelog
:cl:
fix: fixed slimes attacking through windoors
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/slime/life.dm [hunk 1]
```diff
@@ -1,9 +1,11 @@
 /mob/living/simple_animal/slime
 	var/AIproc = 0 // determines if the AI loop is activated
-	var/Atkcool = 0 // attack cooldown
 	var/Discipline = 0 // if a slime has been hit with a freeze gun, or wrestled/attacked off a human, they become disciplined and don't attack anymore for a while
 	var/SStun = 0 // stun variable
 
+	var/monkey_bonus_damage = 2
+	var/attack_cooldown = 0
+	var/attack_cooldown_time = 30 //How long, in deciseconds, the cooldown of attacks is
```
Comment (zeskorion) [inline code/modules/mob/living/simple_animal/slime/life.dm]: so, do slimes attack only once every three seconds now?

Comment (MrLostman) [reply code/modules/mob/living/simple_animal/slime/life.dm]: Yeah it's the same standard for other simple mobs

**Comment by TeomanTheGreat** [comment]
I got killed by a slime attacking through a window today plz merge

File: code/modules/mob/living/simple_animal/slime/life.dm (hunk 1 above)
Comment (zeskorion) [reply code/modules/mob/living/simple_animal/slime/life.dm]: no, other simplemobs have a cooldown of once every two seconds, please fix.

**Review by zeskorion — DISMISSED** [review]

**Review by zeskorion — APPROVED** [review]

**Pull request merged**